src/hotspot/share/oops/symbolHandle.hpp

```diff
@@ -43,16 +44,29 @@
 // probe() and lookup_only() will increment the refcount if symbol is found.
 template <bool TEMP>
 class SymbolHandleBase : public StackObj {
+  static Symbol* volatile _cleanup_delay_queue[];
+  static volatile uint _cleanup_delay_index;
+
   Symbol* _temp;
 
 public:
+  static constexpr uint CLEANUP_DELAY_MAX_ENTRIES = 128;
+
   SymbolHandleBase() : _temp(nullptr) { }
 
   // Conversion from a Symbol* to a SymbolHandleBase.
   // Does not increment the current reference count if temporary.
   SymbolHandleBase(Symbol *s) : _temp(s) {
```

Comment:
Is this really called with nullptr sometimes?  It would be better if that was disallowed.  But that's probably
outside the scope of this PR.